src/map/utils/itemutils.cpp

```diff
@@ -30,6 +30,7 @@
 CItem *		g_pItemList[MAX_ITEMID];    // глобальный массив указателей на игровые предметы
 DropList_t* g_pDropList[MAX_DROPID];    // глобальный массив списков выпадающих предметов
 LootList_t* g_pLootList[MAX_LOOTID];
+DigList_t* g_pDigList[MAX_ZONEID]; // Diglist

```

Comment:
again, not every zone needs a diglist, so there's no reason to use all that memory for each one